modules/ROOT/pages/changes-deprecations-removals.adoc

```diff
@@ -0,0 +1,294 @@
+|<prefix>.cluster.discovery.replicated_data|Size of replicated data structures. (gauge)
+|<prefix>.cluster.discovery.cluster.members|Discovery cluster member size. (gauge)
+|<prefix>.cluster.discovery.cluster.unreachable|Discovery cluster unreachable size. (gauge)
+|<prefix>.cluster.discovery.cluster.converged|Discovery cluster convergence. (gauge)
+|<prefix>.cluster.discovery.restart.success_count|Discovery restart count. (gauge)
+|<prefix>.cluster.discovery.restart.failed_count|Discovery restart failed count. (gauge)
+|===
+====
+
+=== SSL framework
+
+.The CBC-based ciphers that are no longer supported from Neo4j 2025.01:
+
+* TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
+* TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+* TLS_DHE_RSA_WITH_AES_256_CBC_SHA256
+* TLS_DHE_RSA_WITH_AES_128_CBC_SHA256
+
+For more details, refer to the page on link:{neo4j-docs-base-uri}/operations-manual/5/security/ssl-framework/#ssl-other-configs[SSL framework] under the Security section.
+
+.SSL-related configuration setting
+
+Starting from 2025.01, the default value of `dbms.ssl.policy.*.verify_hostname` is changed from `false` to `true`.
+See link:{neo4j-docs-base-uri}/operations-manual/5/security/ssl-framework/#ssl-configuration[the table on the page _SSL framework_] for more details.
+
+=== Ports
+
+Port `5000` is no longer used from 2025.01 onwards.
+
+=== Deprecated built-in procedures
+
+.See all deprecated procedures
+[%collapsible]
+====
+[options=header, cols="3m,1,1,3"]
+|===
+| Name
+| Community Edition
+| Enterprise Edition
+| Comment
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_dbms_quarantineDatabase[`dbms.quarantineDatabase()`] label:admin-only[]
+|
+| {check-mark}
+| Replaced by xref:procedures.adoc#procedure_dbms_unquarantineDatabase[`dbms.unquarantineDatabase()`]
+|===
+====
+
+=== Java API
+
+In 2025.01, the deprecated public API is removed.
+
+The `SeedProvider` is removed and replaced by the `DatabaseSeedProvider`.
+For details, refer to the link:{neo4j-docs-base-uri}/java-reference/current/extending-neo4j/project-setup/#_databaseseedprovider[Java Reference -> Extending Neo4j -> Setting up a plugin project].
+
+=== Neo4j configuration settings with changed default values
+
+[options=header, cols="3m,1,1"]
+|===
+| Name
+| Old default value
+| New default value
+
+|`dbms.cypher.infer_schema_parts`
+|`OFF`
+|`MOST_SELECTIVE_LABEL`
+
+|`db.logs.query.annotation_data_format`
+|`CYPHER`
+|`JSON`
+
+|`dbms.databases.seed_from_uri_providers`
+|`S3SeedProvider,CloudSeedProvider`
+|`CloudSeedProvider`
+
+|`server.metrics.csv.rotation.compression`
+|`NONE`
+|`ZIP`
+
+|`server.panic.shutdown_on_panic`
+|`false`
+|`true`
+
+|===
+
+
+=== Changed valid values of configuration settings
+
+The `server.logs.config` is changed to depend on the provided path for the _conf_ directory instead of having the _conf_ directory encoded in the default values.
```

Comment:
```suggestion
The `server.logs.config` and the `server.logs.user.config` are changed to depend on the provided path for the _conf_ directory instead of having the _conf_ directory encoded in the default values.
```